cosmos/ingestion/ingest/scripts/enrich_parquets.py

```diff
@@ -0,0 +1,26 @@
+from ingest.ingest import Ingest
+from pathlib import Path
+
+cluster = 'tcp://localhost:8786'
+tmp_dir = '/hdd/iain/enrichment_testing/tmp'
+use_semantic_detection = True
```

Comment:
Is there a reason these are set to `True`? Hopefully it doesn't matter, since we're only calling `enrich`, but just a sanity check.

Reply:
They were required args to instantiate an Ingest, but as the ingest() method is never called it's not really doing anything.